crates/transaction-pool/src/ordering.rs

```diff
@@ -31,7 +31,7 @@ pub trait TransactionOrdering: Send + Sync + 'static {
     type PriorityValue: Ord + Clone + Default + fmt::Debug + Send + Sync;
 
     /// The transaction type to determine the priority of.
-    type Transaction: PoolTransaction;
+    type Transaction: PoolTransaction<Pooled = PooledTransactionsElementEcRecovered>;
```

Comment:
cool, this has the same effect but makes it easier to make progress, nice

Reply:
yes seems nice for now for both the comments